src/routes/solid-router/reference/components/memory-router.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: MemoryRouter
+---
+
+The `MemoryRouter` can be used to route while keeping the entire routing history internally in memory.
+Other routers keep the history externally i.e. in the url like the `<HashRouter>` or in the browser history like the [`<Router>`](/solid-router/reference/components/memory-router).
```

Comment:
```suggestion
The `MemoryRouter` can be used to route while keeping the entire routing history within internal memory.
Other routers keep history externally such as in the url like the [`<HashRouter>`](/solid-router/reference/components/hash-router) or in the browser history like [`<Router>`](/solid-router/reference/components/memory-router).
```

Reply:
I can add this link, just note it'll break CI until the HashRouter page also is merged.

Reply:
I'll merge them both when they're good to go :)